src/pages/ResourcePage.test.tsx

```diff
@@ -270,4 +407,75 @@ describe(`<ResourcePage />`, () => {
       );
     });
   });
+
+  it('Should show resource copying opening periods with checkboxes and state change according to interaction, also disabling going to resource batch update page if none selected with  notification', async () => {
```

Comment:
Seems like a lot for single test, perhaps should be separated 🤔

Reply:
Yeah splitting 👍